packages/web/src/components/tracks-table/TracksTable.js

```diff
@@ -39,25 +40,28 @@ export const alphaSortFn = function (a, b, aKey, bKey) {
   return a.toLowerCase() > b.toLowerCase() ? 1 : -1
 }
 
+const allRowsActionButtonRefs = []
```

Comment:
Ideally we'd index these or tie them directly to their own row, but because these are draggable, I wasn't confident that would work consistently. I think this will be low enough overhead since it's only on clicks within the table, but maybe worth thinking about.